src/alg_utils.jl

```diff
@@ -105,6 +105,11 @@ get_current_alg_order(alg::QNDF,cache) = cache.order
 get_current_adaptive_order(alg::QNDF,cache) = cache.order
 get_current_adaptive_order(alg::OrdinaryDiffEqExtrapolationVarOrderVarStepAlgorithm,cache) = cache.cur_order
 get_current_alg_order(alg::OrdinaryDiffEqExtrapolationVarOrderVarStepAlgorithm,cache) = cache.cur_order
+get_current_alg_order(alg::ExtrapolationMidpointDeuflhard,cache) = 2(cache.n_curr + 1)
+get_current_adaptive_order(alg::ExtrapolationMidpointDeuflhard,cache) = 2cache.n_curr + 1
```

Comment:
The error estimator always uses an embedded method of order 1 less? Just double checking since I don't have the book on me for how Hairer defined it.

Reply:
No that is a mistake. The embedded method is of order `2cache.n_curr`. I will change that